modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/freemarker/tool/java/parser/ResourceOpenAPIParser.java

```diff
@@ -808,16 +810,12 @@ private static List<JavaMethodParameter> _getJavaMethodParameters(
 					"application/x-www-form-urlencoded is not supported");
 			}
 			else if (!requestBodyMediaTypes.contains("multipart/form-data")) {
-				RequestBody requestBody = operation.getRequestBody();
-
-				Map<String, Content> contents = requestBody.getContent();
-
-				Iterator<String> iterator = requestBodyMediaTypes.iterator();
-
-				Content content = contents.get(iterator.next());
+				if (schema == null) {
```

Comment:
Maybe this condition could be simplified along with the one in the line #805.

For example:

```
if (requestBodyMediaTypes.contains(
					"application/x-www-form-urlencoded")) {

	throw new RuntimeException(
		"application/x-www-form-urlencoded is not supported");
}

if (!requestBodyMediaTypes.isEmpty() && schema != null) {
    ....
}
```

Reply:
But for example, if I change the position of that condition then the Multipart/form-data can't be introduced with an empty schema. I don't know if creating it without a schema makes sense but maybe is something we have to consider.

Reply:
yes, you are right, it could affect those cases, but I don't think anybody will create a multipart/form-data without a data. At least in Liferay, no REST APIs have the schema empty. We could leave it as it is